server/sonar-process-monitor/src/main/java/org/sonar/application/SchedulerImpl.java

```diff
@@ -121,7 +121,7 @@ private void tryToStartWeb() {
     } else {
       Optional<String> leader = appState.getLeaderHostName();
       if (leader.isPresent()) {
-        LOG.info("Waiting for initialization from " + appState.getLeaderHostName());
+        LOG.info("Waiting for initialization from " + appState.getLeaderHostName().get());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3655)